docs/modules/mapstore/pages/configuring-a-generic-mapstore.adoc

```diff
@@ -348,12 +347,12 @@ YAML::
 ----
 hazelcast:
   map:
-    default:
+    mymapname:
       map-store:
         enabled: true
         class-name: com.hazelcast.mapstore.GenericMapStore
         properties:
-            external-data-store-ref: my-mysql-database
+            data-link-ref: my-mysql-database
```

Comment:
columns are comma separated column names. In this example there is only one column. Maybe give a second column too ? It is up to you